src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -407,11 +410,11 @@ uct_rdmacm_iface_process_event(uct_rdmacm_iface_t *iface,
     case RDMA_CM_EVENT_CONNECT_ERROR:
     case RDMA_CM_EVENT_DISCONNECTED:
         /* Server/Client - connection was disconnected */
-        if (status != UCS_ERR_REJECTED) {
```

Comment:
rdmacm_iface.c is old CM code, no need to change it

Reply:
in case if leave it as-is it produces error message to log and analyser terminates io-demo

Reply:
are you sure it's coming from this file and not rdmacm_cm.c?

Reply:
yes, I found this location from log output

Reply:
maybe UCX_SOCKADDR_CM_ENABLE=y was not set as needed?

Reply:
it could be reason - there are broken infrastructure in reproduction scripts
will double check

Reply:
reverted